src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CodeActionFormattingTest.cs

```diff
@@ -28,17 +28,18 @@ class Goo$$
 ",
 codeActionEdits: new []
 {
-    Edit(7, 17, 7, 17, "Diagnostics;\r\n    using System."),
```

Comment:
These edits were just captured from a debugging session in VS, so I had to adjust the line numbers for the tests, since the generated code for the test data didn't line up with the generated code for whatever random BlazorApp74 I was using when i generated the edits. I didn't bother doing that for this specific edit, because i knew it was going to be thrown away anyway. Now that its not thrown away, I had to do it proper like.